### PR TITLE
Added staled issues bot that closes issues

### PR DESCRIPTION
**Pull request by mojcaostir**

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - security
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: false
```
Comment (goldbergyoni) [inline .github/stale.yml:17]: Should not we leave a comment when closing?

**Review by goldbergyoni — COMMENTED** [review]
@mustamusta Welcome to Practica 🌻

Great contribution! See minor question

**Comment by mojcaostir** [comment]
I`ve updated PR (and changed my username from @mustmusta to @mojcaostir  ;) )

**Review by goldbergyoni — APPROVED** [review]
Awesome!

**Comment by goldbergyoni** [comment]
@mojcaostir Great contribution, this will save us from tedious manual work and let us focus on the important things. Hope to see you here again

@all-contributors please add @mojcaostir for code

**Pull request merged**